src/main/java/lotto/domain/Money.java

```diff
@@ -0,0 +1,37 @@
+package lotto.domain;
+
+import lotto.exception.DivideMoneyException;
+import lotto.exception.ZeroMoneyException;
+
+public class Money {
+    private final int money;
+
+    private static final int ZERO = 0;
+    private static final int THOUSAND = 1000;
```

Comment:
저는 특별한 논리적인 의미를 가지지 않는 상수들은 추출하지 않고 매직 넘버로 그대로 사용하는 편입니다.
만약 해당 값들이 논리적인 의미를 가진다면, 그 의미를 잘 나타낼 수 있도록 네이밍을 해보는 건 어떨까요?
```suggestion
    private static final int LOTTO_PRICE = 1000;
```